src/replica/schema/migrate-None-to-0.sql

```diff
@@ -0,0 +1,660 @@
+
+CREATE USER IF NOT EXISTS qsreplica@localhost;
+CREATE USER IF NOT EXISTS qsreplica@'%';
+
+CREATE DATABASE IF NOT EXISTS qservReplica;
+
+GRANT ALL ON qservReplica.* TO  qsreplica@localhost;
+GRANT ALL ON qservReplica.* TO  qsreplica@'%';
+
+FLUSH PRIVILEGES;
```

Comment:
If I understand correctly the idea of the `None-to-0` stage schema migration the only these lines should stay in this file(?). These are just for initializing the database. Anything else below line 10 needs to be put into the actual schema migration.

My other comment is that for the sake of consistency (and for other good reasons), the account name and the (optional) password need to be injected into the database initialization file as you did it in `src/admin/schema/migrate-None-to-0.sql.jinja`.

This will help to unify and simplify the site-specific initialization of the MySQL instances (czar's, Replication system's and workers').